src/groundlight/client.py

```diff
@@ -95,6 +112,20 @@ def __init__(self, endpoint: Optional[str] = None, api_token: Optional[str] = No
                     ),
                 ) from e
 
+        should_disable_tls_verification = disable_tls_verification or bool(
```

Comment:
Still not right.  We want the parameter's value in code, if set, to take precedence over the environment variable.  Here, if the code sets the value to `False` but the environment variable is true (`1`) then the result will be true.

We should only look at the environment variable if the parameter is `None`.